src/main/scala/service/HelloService.scala

```diff
@@ -1,35 +1,24 @@
 package service
 
-import akka.actor.ActorSystem
-import akka.event.LoggingAdapter
+import akka.http.scaladsl.model.HttpResponse
+import akka.http.scaladsl.model.StatusCodes._
 import akka.http.scaladsl.server.Directives._
-import com.typesafe.config.Config
-
-import scala.concurrent.ExecutionContextExecutor
-
 
 /**
   * Created by joaquim.silveira on 17/05/16.
   */
 trait HelloService {
-  implicit val system: ActorSystem
-  implicit def executor: ExecutionContextExecutor
-
-  def config: Config
-  val logger: LoggingAdapter
 
   val helloRoutes = {
     pathPrefix("hello") {
-      path("world") {
-        get {
+      get {
+        path("world") {
           complete("Hello World!")
-        }
-      } ~
-      path("ops") {
-        get {
-          sys.error("kaput!")
+        } ~
+        path("ops") {
+          complete(HttpResponse(BadRequest, entity = "kaput!"))
```

Comment:
leave the `sys.error("kaput!")` line, it is an example for the global error handler